game-core/src/main/java/games/strategy/engine/framework/startup/mc/ClientModel.java

```diff
@@ -321,10 +321,10 @@ private void startGameInNewThread(final byte[] gameData, final Map<String, INode
     }
     objectStreamFactory.setData(data);
     final Map<String, String> playerMapping = new HashMap<>();
-    for (final String player : playersToNodes.keySet()) {
-      final String playedBy = playersToNodes.get(player);
+    for (final Map.Entry<String, String> stringStringEntry : playersToNodes.entrySet()) {
```

Comment:
I understand that these variable names were probably created by the IDE, but `stringStringEntry` is kind of useless. :smile:  It helps that the value is being assigned to an intermediate variable with a meaningful name.  If a meaningful name for the entry is too hard to construct, then extracting an intermediate variable for the key to preserve the original information might help (i.e. `final String player = stringStringEntry.getKey()`).

And in many other instances within this PR, it would reduce the size of the diff, especially where the key is used multiple times.